src/Extensions/Compatibility/Logger/Category.cs

```diff
@@ -0,0 +1,65 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System;
+
+namespace Microsoft.Omex.Extensions.Compatibility.Logger
+{
+	/// <summary>
+	/// Logging category
+	/// </summary>
+	[Obsolete("In ILogger category provided as generic type parameter for ILogger<T> so Category struct should not be used", false)]
+	public readonly struct Category : IEquatable<Category>
+	{
+		/// <summary>
+		/// Category
+		/// </summary>
+		/// <param name="name">Category name</param>
+		public Category(string name) =>
+			Name = string.IsNullOrWhiteSpace(name)
+				? throw new ArgumentException("Category name must not null or empty", nameof(name))
+				: name;
+
+
+		/// <summary>
+		/// Name
+		/// </summary>
+		public string Name { get; }
+
+
+		/// <summary>
+		/// Indicates whether the current object is equal to another object of the same type
+		/// </summary>
+		/// <param name="other">An object to compare with this object</param>
```

Comment:
Missing `<returns>`

Reply:
Applies throughout this class.

Reply:
It's intentional, I don't think they are adding any value

Reply:
The reason for providing them is when you generate webpages with all of the interface documentation, it becomes easier to read. It's also done by convention when you have some of the documentation listed. By the way, for methods like this, I think `<inheritdoc />` would have worked.

Reply:
You are totally right about `<inheritdoc />` totally forgot about it here.
I understand your concern in general, but I don't like writing parameter comments that just repeat type or parameter name it does not add any value but takes time to maintain, takes useful screen space and also won't users reading documentation.